client/src/components/maps/MyMap.js

```diff
@@ -273,17 +273,24 @@ export const MyMap = () => {
     </div>
 
         <div>
+          <div className="info-window">
         {selectedHike ? (
 
         // InfoWindow to display each hikes attributes. 
-        
+
         <InfoWindow
-          className="info-window"
+          style={{height: "200px"}}
           position={{ "lat": parseFloat(selectedHike.lat), "lng": parseFloat(selectedHike.lng) }}
           onCloseClick={() => {setSelectedHike(null)}}>
           <div className ='pin-description'>
-            <h2>Title - { selectedHike.title } </h2>
-            <p>Description - { selectedHike.description } </p>
+
+            <h3> { selectedHike.title } </h3>
+            <p>{ selectedHike.description } </p>
+            <img src="https://images.fineartamerica.com/images/artworkimages/mediumlarge/2/happy-campers-live-here-unknown.jpg"
```

Comment:
line 289 - 293 need to be deleted since they are appearing below. Otherwise duplication